test/ao/sparsity/test_data_sparsifier.py

```diff
@@ -2,12 +2,152 @@
 # Owner(s): ["module: unknown"]
 
 import logging
-
+import random
+import torch
+from torch.nn.utils.parametrize import is_parametrized
 from torch.testing._internal.common_utils import TestCase
+from torch.ao.sparsity import BaseDataSparsifier
+from typing import Tuple
 
 logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s', level=logging.INFO)
 
 
-class TestBaseDataSparsifier(TestCase):
+class ImplementedSparsifier(BaseDataSparsifier):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def update_mask(self, name, data, **kwargs):
+        mask = self.get_mask(name)
+        mask[0] = 0
+        linear_state = self.state[name]
+        linear_state['step_count'] = linear_state.get('step_count', 0) + 1
+
+
+class TestBaseDataSparsiferType(TestCase):
```

Comment:
I only just now understood whats going on with this test class, I hadn't realized before that this was a helper class for the actual test class TestBaseDataSparsifier. I'm not understanding the name here, feels like it should be something like TestBaseDataSparsifier(Generator/Helper/Runner) or something like that and it should have a comment similar to the comment for the base class describing this.

Reply:
The helper class could take in data of any time and then run tests on them, so I had suffixed with Type. Renamed it to TestBaseDataSparsifierRunner as it seems to be a better name. Added comments as well.